.github/workflows/verify-code-formatting.yml

```diff
@@ -35,4 +28,4 @@ jobs:
           PULL_REQUEST_URL=$(jq -r ".pull_request.url" "$GITHUB_EVENT_PATH")
           curl --url $"${PULL_REQUEST_URL}/files" --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' | \
             jq -c '.[] | select(.status == "added" or .status == "modified") | select(.filename | match("\\.(md|json)$")) | .filename' | \
-            xargs npx prettier@2.0.4 --check
+            xargs -r npx prettier@2.0.4 --check
```

Comment:
Well this is a neat option!

Reply:
What does this do? I can't see `-r` in the docs on my mac's manpage for xargs.

Reply:
It makes it exit gracefully when there's no input, rather than running the command anyway. prettier fails when there's no input.

```
       -r, --no-run-if-empty
              If the standard input does not contain any nonblanks, do not run the command.  Normally, the command is
              run once even if there is no input.  This option is a GNU extension.
```

Reply:
OK, cool. Thanks. So that option doesn't seem to be available on macosx's `xargs`. Maybe `brew` has it's own. Just something to be aware of in general.

Reply:
If it were required, the way to get around not using the option would be to capture the output of the command up to the point before `xargs` then some something like:

```shell
if [[ ! -z $output ]]; then
  # do xargs
fi
```